### PR TITLE
Update Validation-and-Serialization.md

### PR DESCRIPTION
**Pull request by cemremengu**
Explains #1093

#### Checklist
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Validation-and-Serialization.md [hunk 1]
```diff
@@ -109,16 +109,28 @@ The function `getSchemas` returns all shared schemas that were added by `addSche
 
 The `schemaCompiler` is a function that returns a function that validates the body, url parameters, headers, and query string. The default `schemaCompiler` returns a function that implements the `ajv` validation interface. Fastify uses it internally to speed the validation up.
 
-While you cannot change the configuration options of the default `ajv` instance, you can create your own:
+The configuration options of the default fastify `ajv` instance are set as
```
Comment (jsumners) [inline docs/Validation-and-Serialization.md]: "fastify" => "Fastify"

Comment (jsumners) [reply docs/Validation-and-Serialization.md]: In fact, just replace the sentence with:

"Fastify's baseline `ajv` configuration is:"

File: docs/Validation-and-Serialization.md [hunk 2]
```diff
@@ -109,16 +109,28 @@ The function `getSchemas` returns all shared schemas that were added by `addSche
...
+
+```js
+{
+  removeAdditional: true,
+  useDefaults: true,
+  coerceTypes: true
+}
+```
+
+and cannot be changed. If you want to change or set additional config options, you will need to create your own instance and override the existing one like:
```
Comment (jsumners) [inline docs/Validation-and-Serialization.md]: "and cannot be changed." => "This baseline configuration cannot be modified."

**Comment by mcollina** [comment]
@jsumners are you good with this?

**Pull request merged**